Jenkinsfile

```diff
@@ -67,6 +67,43 @@ def buildAndTest(nodeLabel, target, rakeEnv, encrypted) {
     }
 }
 
+def buildAndIamTest(nodeLabel, target, rakeEnv, encrypted) {
```

Comment:
I'd rather this wasn't duplicating most of the code from `buildAndTest` method.
Can't we just surround in an extra `withCredentials` e.g.

```groovy
def buildAndIamTest(nodeLabel, target, rakeEnv, encrypted) {
    withCredentials([string(credentialsId: 'clientlibs-test-iam', variable: 'TEST_COUCH_IAM_API_KEY')]) {
        buildAndTest(nodeLabel, target, rakeEnv, encrypted)
    }
}
```

Reply:
Removed method in 7a2a992.